src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -57,26 +59,33 @@ export default class MxGraphRenderer {
     if (bpmnElementParent) {
       return bpmnElementParent;
     }
-    return this.graph.getDefaultParent();
+
+    if (bpmnElement.kind !== ShapeBpmnElementKind.EVENT_BOUNDARY) {
```

Comment:
For the boundary event, we say its parent is the activity which is attached.
If the activity is not parsed (because we don't support again this kind), I think it's better to not display it and attached it to the parent of the graph.

Reply:
It's not only for sub-processes. 
If there is an error in the BPMN file, where the boundary is attached to an activity who doesn't exist, we have to generate a error and not display it.